src/js/oncoprintruleset.js

```diff
@@ -121,6 +121,7 @@ var makeNAShapes = function(z) {
 };
 var NA_STRING = "na";
 var NA_LABEL = "No data";
+var omitNArule = true;
```

Comment:
whats going on here?

Reply:
Ok, the omitNArule variable assigns readability to use of the boolean switch that controls the suppression of NA-rule addition by GradientRuleSet and CategoricalRuleSet classes. The var is used is the constructor of GradientCategoricalRuleSet:

>function GradientCategoricalRuleSet(params) {
> 	RuleSet.call(this, params);
> 	this.prototype = Object.create(RuleSet.prototype);
>      this.gradientRuleSet = new GradientRuleSet(params);
>      this.categoricalRuleSet = new CategoricalRuleSet(params, **omitNArule**);
>}

would you prefer to keep the var and its use more close to each other (see blow)?
>function GradientCategoricalRuleSet(params) {
> 	RuleSet.call(this, params);
> 	this.prototype = Object.create(RuleSet.prototype);
>      **var omitNArule = true;**
>      this.gradientRuleSet = new GradientRuleSet(params);
>      this.categoricalRuleSet = new CategoricalRuleSet(params, **omitNArule**);
>}

Reply:
yes, I don't think it makes sense to have a variable visible to the entire file that's only used in one specific way. I don't think you need the variable at all, just pass in `true`

Reply:
Ok, I will remove this and just pass a naked boolean.